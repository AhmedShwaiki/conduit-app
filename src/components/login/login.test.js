import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import { React } from "react";
import Login from "./login";
describe("Login componenet", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Login />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
