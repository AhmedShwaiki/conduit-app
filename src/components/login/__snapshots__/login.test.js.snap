// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login componenet renders correctly 1`] = `
<h1>
  Sign in
</h1>
`;
